### PR TITLE
[TTAHUB-1841] Some feedback from session & training report demos

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

- Design asked to adjust margin between cards on training reports and goal dashboard
- Design asked to adjust margin between label and value on the "View" training reports
- Design asked to move the "incomplete form/sessions" message to **above** the submit button group
- Automatically set a training report to "in progress" if you start editing it
- Simplified the success message on the landing page since it was broken (we can iterate on this, but trying to put it in an MVP state)
-  When adding a session, the form should load at the top of the page rather than the scroll position of the previous view.
- Add Back to training reports link at the top of form pages
- When creating a session, the "complete session" section should have a state of "not started" rather than "in progress" before the section has been entered.
- The H2 on the Complete sessions section should say "Complete session" rather than "complete event"
- Update content of success alert when submitting session to "You successfully submitted the session.“
- Update responsive behavior on the Event Cards
- Move "save draft" alert above the buttons (session, event, and AR forms)

## How to test
 
Confirm that the issues brought up by the designers were addressed.
As a sanity check, run through AR, Event, and session creation to make sure that nothing got tweaked (stuff was changed all the places)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1841

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
Converting to draft so I can address the rest of the feedback via this PR

File: frontend/src/pages/TrainingReportForm/index.js [hunk 1]
```diff
@@ -282,7 +289,9 @@ export default function TrainingReportForm({ match }) {
         regionId: regionId || null,
       });
       resetFormData(hookForm.reset, updatedEvent);
+
       updateLastSaveTime(moment(updatedEvent.updatedAt));
+      history.push('/training-reports/complete', { message: 'You successfully submitted the event.' });
```
Comment (AdamAdHocTeam) [inline frontend/src/pages/TrainingReportForm/index.js:294]: Nice so we redirect now.

Comment (thewatermethod) [reply frontend/src/pages/TrainingReportForm/index.js:294]: only because you pointed it out lol

**Review by AdamAdHocTeam — COMMENTED** [review]
Found two weird bugs editing an existing session report (blank):

1. Completing the first section or any subsequent section never changes the status to complete.
2. The recipients list is emtpy.

**Comment by thewatermethod** [comment]
> Found two weird bugs editing an existing session report (blank):
> 
>  1. Completing the first section or any subsequent section never changes the status to complete.
> 
>   2. The recipients list is empty.

@AdamAdHocTeam 

I think something's up - I am able to complete sections, and I have participants. See screenshot below

Screenshot 2023-07-06 at 7 05 12 PM

**Review by AdamAdHocTeam — APPROVED** [review]
Seems the issues I found were related to old session reports that didn't have the region id set.

Overall looks good.

**Pull request merged**